scripts/xapi.conf

```diff
@@ -68,6 +68,9 @@ igd-passthru-vendor-whitelist = 8086
 # kinds of Intel hardware.
 # gvt-g-whitelist = /etc/gvt-g-whitelist
 
+# Mask of CPU features that should not prevent a VM migration
```

Comment:
It's for historical reasons really. I have filled in here what was the `cpuid_default_feature_mask`, which was set for compatibility with XS versions older than 5.6, which had different pool join checks. In any case, this mask needs to be replaced with the real don't care mask, once we know what it is.